Remove error handlers from examples (#1983)
python-telegram-bot/python-telegram-bot: examples/README.md

### [`persistentconversationbot.py`](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/persistentconversationbot.py)
A basic example of a bot store conversation state and user_data over multiple restarts.

### [`errorhandlerbot.py`](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/errorhandlerbot.py)

> A simple example of a bot that has a custom error handler.

Reviewer: ```suggestion
A simple example for a custom error handler.
```

Author: Both the file change name and the addition of start command were implying it's not example of just an error handler (+ minimal code required to run it)

Author: Doesn't mean I'm specifically against this change, I just wanted to point this thing-example vs bot-example parity even earlier, when these two changes were requested.

Reviewer: hmm. The original sentence kinda focuses on the bot part imho, for which we have more suited examples (now). This is mainly for the custom error one. Well, I dont know, it was a mere suggestion, nothing more.

Author: I'm going cast a 0.25 vote against this change. But I'm obviously biased because I wrote the original text. Feel free to hit that "commit suggestion" button if you think it's better.

Reviewer 2: The file name change was simply to fit with the other examples (didn't notice that we three without trailing `bot`, though, but doesn't really matter). I'll just rephrase to `A basic example on how to set up a costum error handler.` without overthinking and merge :D
